cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -0,0 +1,175 @@
+import os
+
+import yaml
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_imagebuilder as imagebuilder
+from aws_cdk import aws_ssm as ssm
+from aws_cdk import core
+
+import pcluster.utils as utils
+from common.utils import load_yaml
+from pcluster.models.imagebuilder import ImageBuilder
+
+
+class ImageBuilderStack(core.Stack):
+    """Create the Stack for imagebuilder."""
+
+    def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBuilder, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+        self.imagebuild = imagebuild
```

Comment:
I don't think you need to set the imagebuild in the stack object.

Reply:
Removed